static/js/message_view_header.js

```diff
@@ -93,47 +92,16 @@ function append_and_display_title_area(message_view_header_data) {
     }
 }
 
-function bind_title_area_handlers() {
```

Comment:
these are the old event handlers for search when there was an icon and search bar, new event handlers are in `search.js`